app/models/post.rb

```diff
@@ -226,6 +226,10 @@ def update_feedback_cache
       SmokeDetector.send_message_to_charcoal "**fp on autoflagged post**: #{title}](//metasmoke.erwaysoftware.com/post/#{id})"
     end
 
+    if is_fp_changed? && is_fp
+      @stack_exchange_user.unblacklist_for_post(self)
```

Comment:
This logic should move to the `Feedback` model in the `after_save` callback.

Reply:
Will do when I'm back on my PC. Also, what's the "fp on autoflagged" thing doing here?